.pre-commit-config.yaml

```diff
@@ -1,4 +1,10 @@
 repos:
+- repo: local
```

Comment:
Is it possible to use the repo `https://github.com/pre-commit/mirrors-eslint`?

Reply:
Seems acceptable. I didn't find this repo before. THX!

Reply:
You can take a look at all the available hooks here: https://pre-commit.com/hooks.html